pkg/binlog-filter/filter.go

```diff
@@ -64,9 +64,41 @@ const (
 	CreateView     EventType = "create view"
 	DropView       EventType = "drop view"
 	AlertTable     EventType = "alter table"
+
+	CreateSchema                  EventType = "create schema" // alias of CreateDatabase
+	DropSchema                    EventType = "drop schema"   // alias of DropDatabase
+	AddColumn                     EventType = "add column"
+	DropColumn                    EventType = "drop column"
+	AddIndex                      EventType = "add index" // alias of CreateIndex
+	AddForeignKey                 EventType = "add foreign key"
+	DropForeignKey                EventType = "drop foreign key"
+	ModifyColumn                  EventType = "modify column"
```

Comment:
It only lists a part of DDL, for example. you missed ALTER TABLE CHANGE COLUMN.

How can we use these new types?

Reply:
1. No, I didn't, all event types TiCDC want to support are aboved.
2. TiCDC's will allow user to specify these new added even type to filter rule.

Reply:
the problem is even you, don't know the the meaning of these types. ALTER TABLE CHANGE COLUMN will generate a "modify column" DDL job, as well as ALTER TABLE MODIFY COLUMN. So we can't count on that user will understand their meaning.

These types derive from TiDB's internal type of ddl jobs, I think configuration should not rely on internal information.